doc/classes/EditorPlugin.xml

```diff
@@ -100,6 +101,9 @@
 			<argument index="0" name="importer" type="EditorImportPlugin">
 			</argument>
 			<description>
+				Registers a new import plugin. Import plugins are used to add a new [Resource] which can be imported. See [EditorImportPlugin] for more information.
```

Comment:
I think all the "See `ClassName` for more information" can be removed and instead the opening sentence can be formatted as "Register a new `ClassName`". This includes the existing description for `add_export_plugin` which you used as an example for your new descriptions.

```suggestion
				Registers a new [EditorImportPlugin]. Import plugins are used to import custom and unsupported assets as a custom [Resource] type.
```